File: RA1/clase12/JAVA/Gato.java
public class Gato extends Animal{
    private double flexibilidad;

    public Gato(String nombre, String especie, String raza, int edad, double peso, double flexibilidad){
        super(nombre, especie, raza, edad, peso); // es llamar al constructor de la super clase
        this.flexibilidad = flexibilidad;
    }

    public void purr(){
        System.out.println(this.nombre + " esta ronrroneando...");
    }

    //Metodo por sobrecarga
    public void sleep(){
        System.out.println(this.nombre + " es un gato que duerme... zzz ");
    }

    @Override
    public void makeSound(){
        System.out.println(this.nombre + " hace miauu...");
    }

}

File: RA1/clase12/PYTHON/Gato.py
from Animal import Animal

class Gato(Animal):
    def __init__(self, nombre, especie, raza, edad, peso, flexibilidad):
        super().__init__(nombre, especie, raza, edad, peso)
        self.flexibilidad = flexibilidad

    def purr(self):
        print(self.nombre + " esta ronrroneando...")

    def sleep(self):
        print(self.nombre + " es un gato que esta duermiendo.. zzz")

    
    def make_sound(self):
        print(self.nombre + " hace un miauu...")
    